### DISCUSSION TITLE
Details of BUDGET regridding method

### DISCUSSION THREAD

**Discussion by scole-mo** [dtcenter/METplus#2643] [Statistical Computation] [answered]
Hello, 

I'm planning on running GridDiag to analyse some precipitation forecasts. I was planning to use the 'budget' regrid method but can't seem to find details of how this actually works - only that mass is conserved as per the [MET documentation](https://met.readthedocs.io/en/latest/Users_Guide/config_options.html#regrid) . Is a more detailed explanation/methodology available?

**Comment by JohnHalleyGotway** [comment, marked as answer]
@scole-mo thanks for pointing out the lack of sufficient detail in the MET User's Guide about the budget interpolation method. I took the opportunity to update the details of the `regrid` section in the develop branch:
https://met.readthedocs.io/en/develop/Users_Guide/config_options.html#regrid
<img width="818" alt="Screen Shot 2024-07-23 at 11 28 39 AM" src="https://github.com/user-attachments/assets/f055b3d9-a062-4550-9ce0-89ae3d89dfc1">

You can find the source code that actually implements this logic in [vx_regrid_budget.cc](https://github.com/dtcenter/MET/blob/main_v11.1/src/libcode/vx_regrid/vx_regrid_budget.cc).

Please let me know if you have any additional questions.

**Reply by scole-mo** [reply]
Thank you @JohnHalleyGotway, much appreciated!
